templates/paragraphs/paragraph--oe-text-feature-media.html.twig

```diff
@@ -15,7 +15,7 @@
 {{ pattern('text_featured_media', {
```

Comment:
Looks like we need to override `text_featured_media` pattern, so there will be values:
* title
* text
* media_container (it is instance of media_container pattern).